onnx/defs/shape_inference.h

```diff
@@ -61,6 +60,9 @@ struct InferenceContext {
   virtual TypeProto* getOutputType(size_t index) = 0;
   virtual GraphInferencer* getGraphAttributeInferencer(const std::string& attribute_name) = 0;
   virtual ~InferenceContext() {}
+  /// XXX: Putting at the end of vtable in case it matters?
+  /// XXX: Do we need default implementation here?
```

Comment:
no we dont... you can remove this comment

Reply:
I think a default implementation does help users who have an implementation of InferenceContext use this version without modifying their code (which seems reasonable if they do not want to handle sparse data yet).